api/v1alpha1/etcd_types_test.go

```diff
@@ -104,15 +104,24 @@ func getEtcd(name, namespace string) *Etcd {
 	prefix := "etcd-test"
 	garbageCollectionPolicy := GarbageCollectionPolicy(GarbageCollectionPolicyExponential)
 
-	tlsConfig := &TLSConfig{
+	clientTlsConfig := &TLSConfig{
+		TLSCASecretRef: corev1.SecretReference{
+			Name: "client-url-ca-etcd",
+		},
 		ClientTLSSecretRef: corev1.SecretReference{
-			Name: "etcd-client-tls",
+			Name: "client-url-etcd-client-tls",
 		},
 		ServerTLSSecretRef: corev1.SecretReference{
-			Name: "etcd-server-tls",
+			Name: "client-url-etcd-server-tls",
 		},
+	}
+
+	peerTlsConfig := &TLSConfig{
 		TLSCASecretRef: corev1.SecretReference{
-			Name: "ca-etcd",
+			Name: "peer-url-ca-etcd",
+		},
+		ServerTLSSecretRef: corev1.SecretReference{
+			Name: "peer-url-etcd-server-tls",
 		},
 	}
```

Comment:
Does peerTlsConfig not seed a `ClientTLSSecretRef`?

Reply:
Right. Currently, there is no need for client certificates for peer connection. Let me add a comment there as well.